releng/org.eclipse.n4js.targetplatform/N4JS.setup

```diff
@@ -25,270 +26,273 @@
+			targetURL="${workspace.location|uri}/.metadata/.plugins/org.eclipse.jdt.ui/dialog_settings.xml"
+			encoding="UTF-8">
+		<description>Initialize JDT's package explorer to show working sets as its root objects</description>
+	</setupTask>
+	<setupTask
+			xsi:type="setup:VariableTask"
+			name="eclipse.target.platform"
+			value="None"
+			storageURI="scope://Workspace"/>
+	<setupTask
+			xsi:type="setup:VariableTask"
+			name="n4js.target.platform"
+			defaultValue="N4JS_Neon.2.IDE-2686"
+			storageURI="scope://Workspace">
+		<choice
+				value="N4JS_Neon.2.IDE-2686"
+				label="N4JS_Neon.2.IDE-2686"/>
+	</setupTask>
+	<setupTask
+			xsi:type="setup.p2:P2Task"
+			label="Platform">
+		<requirement
+				name="org.eclipse.platform.feature.group"/>
+		<requirement
+				name="org.eclipse.rcp.feature.group"/>
+		<requirement
+				name="org.eclipse.jdt.feature.group"/>
+		<requirement
+				name="org.eclipse.pde.feature.group"/>
+	</setupTask>
+	<setupTask
+			xsi:type="setup.p2:P2Task"
+			label="Xcore">
+		<requirement
+				name="org.eclipse.emf.ecore.xcore.sdk.feature.group"
+				versionRange="[1.4.0,1.4.1)"/>
+		<repository
+				url="http://download.eclipse.org/modeling/emf/emf/updates/2.12"/>
+		<description>Install the tools needed in the IDE to work with the source code for ${scope.project.label}</description>
+	</setupTask>
+	<setupTask
+			xsi:type="setup.p2:P2Task"
+			label="Xtext/Xtend">
+		<requirement
+				name="org.eclipse.xtend.sdk.feature.group"
+				versionRange="[2.12.0,2.12.1)"/>
+		<requirement
+				name="org.eclipse.xtext.sdk.feature.group"
+				versionRange="[2.12.0,2.12.1)"/>
+		<repository
+				url="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.12.0"/>
+		<description>Install the tools needed in the IDE to work with the source code for ${scope.project.label}</description>
+	</setupTask>
+	<setupTask
+			xsi:type="setup.p2:P2Task"
+			label="XSemantics">
+		<requirement
+				name="it.xsemantics.sdk.feature.group"
+				versionRange="[1.12.0,1.12.9)"/>
+		<repository
+				url="http://dl.bintray.com/lorenzobettini/xsemantics/updates/1.12"/>
```

Comment:
as far as I can tell, you changed the protocol from `https` to `http`, here. On purpose?